src/components/form/templates/object-field.ts

```diff
@@ -1 +1,63 @@
-export const ObjectFieldTemplate = undefined; // TODO implement object field template
+import React from 'react';
+import { ObjectFieldProperty, ObjectFieldTemplateProps } from './types';
+
+export const ObjectFieldTemplate = (props: ObjectFieldTemplateProps) => {
+    const id = props.idSchema.$id;
+    if (id === 'root' || !props.schema.collapsible) {
```

Comment:
`collapsible` is not part of the JSON schema specification, I just made it up here as an example of how we could display object fields. 
I think it would be neat to have the possibility to have collapsible fields, and I think we might need it for array items as well in order to render them in a nice way.
I'm not sure if this is the correct way to do this though... The spec allows custom properties to be specified in the schema, but it is recommended that a meta schema is specified as well if custom properties are added. I think we should discuss how to handle this properly

Reply:
👍

Reply:
Yeah, collapsible sections will be very nice to have, I think. I wonder if we might want to make the collapsible and non-collapsible ones look more similar in the UX, since the schemas for them are identical apart from the `collapsible: true` property?

I guess we don't have to decide that in this PR though.

Reply:
Approved, but if you feel we should discuss this, I guess it's probably best to do that before merging… 🙂

Reply:
AFK discussion: We'll put our own properties in a dict called `lime`, effectively prefixing them, like this: `props.schema.lime.collapsible`

This is the fastest way to secure the code against name conflicts with potential future standard properties.